### PR TITLE
Replaced jsonfield2 with jsonfield

### PR DESCRIPTION
**Pull request by iamsobanjaved**
**Description:** We are going to replace jsonfield2 with jsonfield as the earlier one has archived and merged back in the latter one.

**JIRA:** https://openedx.atlassian.net/browse/BOM-2446

**Dependencies:** dependencies on other outstanding PRs, issues, etc.

**Merge deadline:** List merge deadline (if any)

**Installation instructions:** List any non-trivial installation
instructions.

**Testing instructions:**

1. Open page A
2. Do thing B
3. Expect C to happen
4. If D happened instead - check failed.

**Merge checklist:**
- [ ] All reviewers approved
- [ ] CI build is green
- [ ] Version bumped
- [ ] Changelog record added
- [ ] Documentation updated (not only docstrings)
- [ ] Commits are squashed

**Post merge:**
- [ ] Create a tag
- [ ] Check new version is pushed to PyPI after tag-triggered build is
      finished.

**Author concerns:** List any concerns about this PR - inelegant
solutions, hacks, quick-and-dirty implementations, concerns about
migrations, etc.

### CODE REVIEW

File: test_settings.py [hunk 1]
```diff
@@ -38,8 +38,6 @@ def root(*args):
     root('event_routing_backends', 'conf', 'locale'),
 ]
 
-ROOT_URLCONF = 'event_routing_backends.urls'
```
Comment (iamsobanjaved) [inline test_settings.py:41]: This module is deleted in this commit, had to remove this to run `makemigrations` command.

https://github.com/edx/event-routing-backends/commit/5531a185e49e356873b69cb0c58c4b0526c0318b#diff-f1b972652ffca6892272e92f8e805afa88f960473f084e6e07015dee212515d0

**Review by UsamaSadiq — APPROVED** [review]

**Comment by iamsobanjaved** [comment]
We had to merge all jsonfield PRs in edX packages at once because this PR will bring latest version of jsonfield in platform as well which needs migration in few other packages as well.

**Comment by jinder1s** [comment]
Is this a change that should require a version bump for event-routing-backends?

**Comment by iamsobanjaved** [comment]
> Is this a change that should require a version bump for event-routing-backends?



> Is this a change that should require a version bump for event-routing-backends?

Yup, just updated the version

**Review by jinder1s — APPROVED** [review]

**Pull request merged**